src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Email/DkimConfigurationType.php

```diff
@@ -38,31 +39,48 @@
  */
 class DkimConfigurationType extends TranslatorAwareType
 {
+    public const FIELD_MAIL_DKIM_DOMAIN = 'domain';
+    public const FIELD_MAIL_DKIM_SELECTOR = 'selector';
+    public const FIELD_MAIL_DKIM_KEY = 'key';
+
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('domain', TextType::class, [
+            ->add(self::FIELD_MAIL_DKIM_DOMAIN, TextType::class, [
```

Comment:
```suggestion
            ->add(self::FIELD_MAIL_DKIM_DOMAIN, TextType::class, [
```

Please, replace `self` by `static`.